### DISCUSSION TITLE
Set amount of images to download

### DISCUSSION THREAD

**Discussion by johnnyp92** [Confuzu/CivitAI_Image_grabber#28] [Ideas]
First up, thanks for this program! it's amazing, does exactly what I need!

Any chance to get a feature where we can set the amount of images to download? Rather than downloading like 5000 images. 

Additionally, I found a way to download via most reactions. Is there a chance to get this kind of option in the command prompt?

Again, thanks!

**Comment by Confuzu** [comment]
check the newest update
